include/deal.II/non_matching/mapping_info.h

```diff
@@ -545,7 +560,8 @@ namespace NonMatching
      * Resize the mapping data fields.
      */
     void
-    resize_data_fields(const unsigned int n_data_point_batches);
+    resize_data_fields(const unsigned int n_data_point_batches,
+                       const bool         is_face_centric = false);
```

Comment:
Isn't this something that is determined during `reinit()`?

Reply:
Which `reinit` do you mean? It depends on which `reinit_faces` is called (ECL or FCL storage), how the data fields are resized.